package.json

```diff
@@ -85,11 +85,11 @@
     "@cypress/react": "5.12.4",
     "@cypress/webpack-dev-server": "1.8.3",
     "@fluentui/react-icons": "^2.0.175",
-    "@griffel/babel-preset": "^1.3.5",
+    "@griffel/babel-preset": "1.3.6",
```

Comment:
Why is this no longer a caret?

Reply:
AFAIR we don't use carets for `devDependencies`